packages/larsoft/package.py

```diff
@@ -137,14 +137,11 @@ def setup_dependent_build_environment(self, spack_env, dspec):
         spack_env.append_path("FW_SEARCH_PATH", "{0}/fw".format(self.prefix))
         sanitize_environments(spack_env)
 
-    def setup_dependent_run_environment(self, run_env, dspec):
-        run_env.prepend_path("PATH", self.prefix.bin)
-        run_env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
+    def setup_run_environment(self, run_env):
         run_env.prepend_path("PATH", self.prefix.bin)
         run_env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
         run_env.append_path("FHICL_FILE_PATH", "{0}/fcl".format(self.prefix))
-        run_env.append_path("FW_SEARCH_PATH", "{0}/fw".format(self.prefix))
-        sanitize_environments(run_env)
+        run_env.append_path("CET_PLUGIN_PATH", self.prefix.lib)
```

Comment:
As `larsoft` is an umbrella product, it provides no FHiCL files or C++ code.  I'd therefore keep only the adjustment for `PATH`.

Reply:
Done.